icaruscode/Filters/FilterCRTPMTMatching_module.cc

```diff
@@ -311,9 +313,11 @@ bool icarus::crt::FilterCRTPMTMatching::filter(art::Event &e)
 
     auto const& crtHitList = e.getProduct<std::vector<CRTHit>>(fCrtHitModuleLabel);
 
-    bool Filter = false;
+    bool Cosmic = false;
     int  Type   = 9;
 
+    if (fFilterLevel != "loose" || fFilterLevel != "tight" || fFilterLevel != "medium") fFilterLevel="loose";
```

Comment:
This part should be in the constructor, since it will happen only once.
And I recommend that the default value of the parameter be changed from `"FilterOption"` to `"loose"`.
I also recommend that if an unknown option is passed, an exception is thrown, and otherwise that a message is printed saying that the filter option was not recognised and therefore we are using `"loose"`. Otherwise a user using by mistake `"Medium"` or `"tight "` will end up with the wrong level without ever noticing it.

And actually my real suggestion is, surprise surprise, to use another enumerator:
```cpp
enum class FilterLevel {
  loose,
  medium,
  right,
  defLevel = loose
};
```
which can be parsed like:
```cpp
  std::string const& filterOptStr = p.get<std::string>("FilterLevel", "loose");
  if     (filterOptStr == "loose") fFilterLevel = FilterLevel::loose;
  else if (filterOptStr == "medium") fFilterLevel = FilterLevel::medium;
  else if (filterOptStr == "tight") fFilterLevel = FilterLevel::tight;
  else throw art::Exception{ art::errors::Configuration } << "Filter level not supported: '" << filterOptStr << "'\n";
```